### DISCUSSION TITLE
Improvement to unify fragments with progress bar

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#585] [Proposals]
In the component Cards/Cards Slots/Example > fragment: progress
We need to add a text to the right above the progress bar to use the same component for progress un data consumption and in devices payment.
Expected result:
<img width="375" alt="image" src="https://user-images.githubusercontent.com/95479574/177489738-43647b07-6b06-4898-9ee9-c96cd7913327.png">

**Comment by yceballost** [comment]
Make sense! we will do it for the next release [5.1.0](https://github.com/Telefonica/mistica-design/milestone/4)
